doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -3028,6 +3028,25 @@ headers are encapsulated in the same number of headers.
 Advanced Options
 ----------------
 
+entropy
+~~~~~~~
+
+When a rule causes an entropy value to be calculated for a flow, output for the flow will include
+the calculated entropy value. The log output contains the sticky buffer name for which the
+entropy was calculated. Often, more context is needed and the configuration setting shown
+below will amend the sticky buffer name with the signature id of the rule that caused
+the entropy value calculation. Additionally, if multiple ``entropy`` keywords are used
+within a rule, the occurrence number is included in the log output.
+The default value is ``on``. We strongly recommend changing using the default value of ``on``.
```

Comment:
I'm a bit confused. Strongly recommend turning it off? Disabling, maybe?

Reply:
Will clarify.